solutions/mostCommon.js

```diff
@@ -0,0 +1,23 @@
+const findMax=(obj,i=0,objKeys=Object.keys(obj),maxKey=objKeys[i])=>{
```

Comment:
can you please require the previous problem and use that instead of repeating the same code again here?